src/components/NavBar.tsx

```diff
@@ -94,6 +94,7 @@ export default function NavBar({
           closeAndNavigateTo(APP_PATHS.INVENTORY);
         }}
         id='inventory'
+        key='inventory'
```

Comment:
What is this key for?

Reply:
This is an array of nav items, react was complaining the elements needed keys.

Reply:
Thanks!